src/components/manage/SubscriptionsPanel/subscriptions-panel.css

```diff
@@ -0,0 +1,54 @@
+#page-subscriptions .ui.button.labeled i.icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+
+#page-subscriptions .ui.checkbox.toggle label {
+    overflow: hidden;
+}
+
+
+#page-subscriptions .ui.form.search-form .ui.fluid.input {
+    justify-content: flex-end;
+    margin-bottom: 0.5rem;
+}
+
+#page-subscriptions .ui.form.search-form .ui.fluid.input input {
+    max-width: 23rem;
+}
+
+#page-subscriptions .ui.message.selected-items {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+#page-subscriptions .no-results {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 1.5rem 1rem;
+    border-right: 1px solid #c7d5d8;
+    border-left: 1px solid #c7d5d8;
+    margin: 0em -1px;
+    font-style: italic;
+    font-weight: 300;
+}
+
+.confirm-delete-item:not(:last-of-type) {
+    margin-bottom: 0.75rem;
+}
+
+.confirm-delete-item {
+    font-weight: 600;
+}
+
+.ui.dimmer .ui.workaround.loader:before {
+    border-color: rgba(0, 0, 0, 0.1);
+}
+
+.ui.dimmer .ui.workaround.loader:after {
+    border-color: #767676 transparent transparent;
+}
```

Comment:
This css looks like it's repeated as in send-history. It can be repeated since it's a small amount but needs at least to be specific to the page where you use it otherwise it applies to everything on the website as soon as you load it the first time

Reply:
yes it's repeated..i made them more specific